src/core/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "strictNullChecks": true,
     "lib": ["dom", "esnext", "es6"],
     "paths": {
+      "#core/assert": ["./assert"],
```

Comment:
nit: does this work:
```
'#core': ['.']
'#core/*': ['./*']
```

Reply:
It will once `context` is allowed (yes we could exclude `context` directly instead). That's where we'll be very soon.